src/Sql/Vault/dbo/Functions/UserCipherDetails.sql

```diff
@@ -54,6 +59,7 @@ SELECT
     *,
     1 [Edit],
     1 [ViewPassword],
+    1 [Manage],
```

Comment:
⛏️ (non-blocking): Tiny nit, but the order of these selected properties are not in the same order for both of the queries in the `UNION ALL`.  (`Edit, Manage, ViewPassword` VS. `Edit, ViewPassword, Manage`)

Reply:
Reordered the input parameter